.github/workflows/tests.yaml

```diff
@@ -80,5 +80,7 @@ jobs:
 
       - run: |
           npm install --global yarn
+          npm install --global node-gyp@9.0.0
+          npm prefix -g | % {npm config set node_gyp "$_\node_modules\node-gyp\bin\node-gyp.js"}
```

Comment:
I'm curious what went wrong here.

Reply:
Because in CI GH we are using `windows-latest` which implies `Microsoft Windows Server 2022` which implies VS2022, and `npm` is using `node-gyp` version `5.1.0` which does not support VS202.
More info: [Upgrading node-gyp independently of npm and handling multiple versions](https://github.com/nodejs/node-gyp/issues/2272)

The other option is to use an older windows version which has a MS VS version compatible with `node-gyp 5.1.0`

Reply:
Ah, looks like they finally switched `windows-latest` over to 2022 :+1: Cool.